pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -1,26 +1,31 @@
 {
-  lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonOlder,
-  pytestCheckHook,
+  fetchpatch,
+  lib,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   gdcm,
   nibabel,
   numpy,
+  pillow,
   pydicom,
-  pylibjpeg,
-  pylibjpeg-libjpeg,
   scipy,
-  setuptools,
+
+  # tests
+  pylibjpeg-libjpeg,
+  pylibjpeg,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
```

Comment:
Final nitpick from me would be to order the arguments to `buildPythonPackage` as follows (as we I think I now settled on which arguments to use):

`pname`, `version`, `pyproject`, `src`, `patches`, `postPatches`, `build-system`, `dependencies`, `pythonImportsCheck`, `nativeCheckInputs`, `meta`.

As this is the order in which `buildPythonPackage` makes use of the arguments passed to it.